### PR TITLE
New Profile Card

### PR DESCRIPTION
**Pull request by lazy-god**
image

image

### CODE REVIEW

**Comment by vdvibhu20** [comment]
`.o-profile {
  justify-content: space-evenly;
  font-size: 22px;`

font-size set to 22px for whole page, its 22 for heading only

**Comment by vdvibhu20** [comment]
screen shot 2018-06-13 at 1 41 09 pm
screen shot 2018-06-13 at 1 43 25 pm

this section is not same as in designs

**Comment by vdvibhu20** [comment]
links are not circular

screen shot 2018-06-13 at 1 45 07 pm
screen shot 2018-06-13 at 1 49 57 pm

**Comment by vdvibhu20** [comment]
remove comments

`//@include margin(0, 80px, 0, 0);`

**Comment by lazy-god** [comment]
image
image

File: sass/styles/objects/o-profile-page/o-profile-page.scss [hunk 1]
```diff
@@ -1,48 +1,105 @@
...
+
+  .faded {
     color: $default-accordian-font;
   }
 
-  .button-circle{
-    @include margin(0, 0, 23px, 0);
+  .auth-link {
+    @include margin(0, 10%, 0, 0);
+    text-align: center;
```
Comment (vdvibhu20) [inline sass/styles/objects/o-profile-page/o-profile-page.scss]: remove this as already specified in elements

File: sass/styles/objects/o-profile-page/o-profile-page.scss [hunk 2]
```diff
@@ -1,48 +1,105 @@
...
   }
 
-  .flex-row{
+  .links {
+    @include padding-mixin (30px, 0, 20px, 0);
+  }
+
+  .flex-row {
+    @include padding-mixin(30px, 0, 30px, 0);
+    display: flex;
```
Comment (vdvibhu20) [inline sass/styles/objects/o-profile-page/o-profile-page.scss]: no need to right `display: flex`

File: sass/styles/media-query/media.card.layout.scss [hunk 3]
```diff
@@ -51,6 +51,11 @@
     max-width: 100%;
   }
 
+  .card-image {
+    float: left;
+    margin-right: 15px;
+  }
+
```
Comment (vdvibhu20) [inline sass/styles/media-query/media.card.layout.scss]: remove this part

**Comment by vdvibhu20** [comment]
screen shot 2018-06-15 at 4 32 07 pm
provide space between heading and button

**Pull request closed without merging**